pkgs/development/libraries/libmesode/default.nix

```diff
@@ -1,32 +1,33 @@
-{ stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, libtool, openssl, expat, pkgconfig, check }:
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
```

Comment:
```suggestion
```

Reply:
Marked as broken and also removed `fetchpatch`, thanks for picking up on that :)